test/EndToEndTests/Tests/Client/Build.Desktop/PluggableFormatServiceTests/PluggableFormatServiceTests.cs

```diff
@@ -287,14 +288,16 @@ public void InvokeAvroAction()
 
         private static void SetAvroMediaTypeResolver(HttpWebRequestMessage requestMessage)
         {
-            requestMessage.Container = ContainerBuilderHelper.BuildContainer(builder =>
-                builder.AddService<ODataMediaTypeResolver, AvroMediaTypeResolver>(ServiceLifetime.Singleton));
+            IServiceCollection services = new ServiceCollection();
+            IServiceProvider serviceProvider = services.AddDefaultODataServices().BuildServiceProvider();
+            requestMessage.Container = ServiceProviderHelper.BuildServiceProvider(builder =>

```

Comment:
Looks like `requestMessage.Container` needs to be changed to `requestMessage.ServiceProvider`.